### ISSUE TITLE
Training with vietnamese dataset. Learning rate = 0 

### ISSUE THREAD

**Issue by ATtendev** [JarodMica/ai-voice-cloning#66]
Dataset 600k sample
Duration 3-10s
After 10 epochs 

![image](https://github.com/JarodMica/ai-voice-cloning/assets/132933484/dab80ecf-9294-4a87-a4ff-07199b1a9e0a)

The learning rate is approximately 0, is that normal?
